compiler/control/OMROptions.hpp

```diff
@@ -1344,6 +1344,7 @@ class OMR_EXTENSIBLE Options
          _optionSets(0),
          _logListForOtherCompThreads(NULL)
       {
+      memset(self(), 0, sizeof(OMR::Options));
```

Comment:
@0xdaryl when you have a chance could you review and merge? The change being made is very simple:

Reply:
Looks ok me. The only danger is that if Options ever uses inheritance in the future, this operation will overwrite the vtable. However, the same problem will be present for jitCmdLineOptions.